business/openshift_oauth.go

```diff
@@ -56,6 +57,8 @@ type OAuthRouteTLSSpec struct {
 
 const serverPrefix = "https://kubernetes.default.svc/"
 
+var defaultRequestTimeout, _ = time.ParseDuration("10s")
```

Comment:
`10*time.Second` would avoid parsing a string (is there any advantage to parse here?)

Reply:
Is there a disadvantage?   I see it as very readable and it is only done one time.

Reply:
I decided there is an advantage, I can use const as opposed to var.  Made the change.